### PR TITLE
Added read_info to docs.

### PR DESCRIPTION
**Pull request by nfoti**
Addresses issue #3507.

### CODE REVIEW

File: doc/python_reference.rst [hunk 1]
```diff
@@ -184,6 +186,15 @@ Functions:
    write_surface
    write_trans
 
+.. currentmodule:: mne.io
+
+:py:mod:`mne.io`:
```
Comment (jasmainak) [inline doc/python_reference.rst]: why do you have this again? did you check how it rendered?

https://2220-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/python_reference.html

**Comment by Eric89GXL** [comment]
If we're going to make it public we should probably move it to the mne
namespace, too, since it's useful for more than raw io (what the mne.io is
publicly meant for)

**Comment by agramfort** [comment]
I agree with @jasmainak comment's about misplaced mod py

@Eric89GXL maybe take over. I am fine adding the read_info function in mne namespace

**Comment by nfoti** [comment]
Ok, I'm going to hand this over to whomever can do it the easiest.

**Comment by agramfort** [comment]
fixed in master

thx @nfoti

**Pull request closed without merging**